server/api/routers/mariadb.ts

```diff
@@ -49,6 +49,9 @@ export const mariadbRouter = createTRPCRouter({
 
 				return true;
 			} catch (error) {
+				if (error instanceof TRPCError) {
```

Comment:
fix：Error messages are not directly thrown to prevent server error sensitive information from appearing on the client